### ISSUE TITLE
Callback implementation doesn't support class#method

### ISSUE THREAD

**Issue by cybey** [breamware/sidekiq-batch#3]
Following the specification on https://github.com/mperham/sidekiq/wiki/Batches under the "Callbacks" header, there is the following example for callbacks.

```ruby
batch.on(:complete, 'AnotherClass#method', 'uid' => current_user.id)
```

This behaviour does not seem supported by sidekiq-batch.

**Comment by managr** [comment]
That's a good catch however the implementation does not distinguish the event that'll be fired. Running:
```ruby
batch.on(:complete, 'AnotherClass#method', 'uid' => current_user.id)
```
will run `method` on both `complete` and `success`.

**Comment by managr** [comment]
Closing this one, since it was released in `0.1.2`
https://github.com/breamware/sidekiq-batch/releases/tag/v0.1.2

Thanks a lot @cybey
